### PR TITLE
Add missing translations

### PR DESCRIPTION
**Pull request by ndrluis**
I added some translations that were missing but now I need some help to fill the others locales.

### CODE REVIEW

File: app/views/administrate/application/edit.html.erb [hunk 1]
```diff
@@ -15,13 +15,13 @@ It displays a header, and renders the `_form` partial to do the heavy lifting.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Page/Form
 %>
 
-<% content_for(:title) { "Edit #{page.page_title}" } %>
+<% content_for(:title) { "#{t("administrate.actions.edit")} #{page.page_title}" } %>
```
Comment (zamith) [inline app/views/administrate/application/edit.html.erb]: What about passing the page title as an option to the translation? That would be good for languages where the title makes more sense before the edit word. What do you think?

Comment (ndrluis) [inline app/views/administrate/application/edit.html.erb]: :+1:

**Comment by graysonwright** [comment]
A big :+1:. Thanks!

File: app/views/administrate/application/edit.html.erb (hunk 1 above)
Comment (graysonwright) [inline app/views/administrate/application/edit.html.erb]: Agreed, something like `t("administrate.actions.edit_resource", resource: page.page_title)` would be good.

**Comment by neves** [comment]
Can you translate `placeholder="Search"` at:
https://github.com/thoughtbot/administrate/blob/master/app/views/administrate/application/index.html.erb#L37

**Comment by hudsonsferreira** [comment]
please @ndrluis just keep the good work here

**Comment by nickcharlton** [comment]
@ndrluis I'd love to merge this. Could you rebase on `master` or [enable contributor pushes](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/)?

**Comment by ndrluis** [comment]
@nickcharlton I enabled the pushes.

**Pull request merged**